lms/views/api/exceptions.py

```diff
@@ -83,13 +83,20 @@ class ExceptionViews:
     def __init__(self, context, request):
         self.context = context
         self.request = request
+        self.default_response_data = {}
+
+        if request.matched_route.name.startswith("canvas_api."):
+            self.default_response_data["auth_url"] = request.route_url(
+                "canvas_api.authorize"
+            )
```

Comment:
In future there'll be an `elif request.matched_route.name.startswith("blackboard_api.")` here